examples/pytest/dummy/conftest.py

```diff
@@ -0,0 +1,69 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+# Code generated by "pykiso_to_pytest". DO NOT EDIT.
+
+
+import logging
+
+import pytest
+
+from pykiso.lib.auxiliaries.dut_auxiliary import DUTAuxiliary
+from pykiso.lib.auxiliaries.example_test_auxiliary import ExampleAuxiliary
+from pykiso.lib.connectors.cc_example import CCExample
+
+
+@pytest.fixture(scope="session")
+def aux1():
+    try:
+        com = CCExample(name="chan1")
+        aux1 = ExampleAuxiliary(
+            com=com,
+        )
+        aux1.start()
+        aux1.create_instance()
+    except Exception:
+        logging.exception("something bad happened")
+    yield aux1
+    aux1.delete_instance()
+    aux1.stop()
+
+
+@pytest.fixture(scope="session")
+def aux2():
+    try:
+        com = CCExample(name="chan2")
+        flash = CCExample(name="chan3")
```

Comment:
Again, this is wrong, we have one additional parameter that is kind of ignored here: https://github.com/eclipse/kiso-testing/blob/master/examples/dummy.yaml#L26